src/Card/Card.js

```diff
@@ -1,12 +1,79 @@
+import { useState } from 'react';
 import cl from 'classnames';
 import './Card.scss';
+import { BiEditAlt, BiXCircle, BiSave } from 'react-icons/bi';
 
-function Card({ className, caption, content, onSelect, selected = false }) {
+function Card({ className, caption, content, onSelect, onEdit,
+                editing = false, selected = false }) {
+
+  // New values state
+  const [state, setState] = useState({ caption, content });
```

Comment:
Can you please rename state value and its setting method?
We don't know sense of "state" name.

Reply:
Will 'cardState' and 'setCardState' be suitable here?